exe/cli/cmd/ffs_getcidconfig.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/caarlos0/spin"
+	"github.com/ipfs/go-cid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	ffsGetCidConfigCmd.Flags().StringP("token", "t", "", "FFS auth token")
+
+	ffsCmd.AddCommand(ffsGetCidConfigCmd)
+}
+
+var ffsGetCidConfigCmd = &cobra.Command{
+	Use:   "getCidConfig [cid]",
```

Comment:
Idea: maybe just calling this `config`? e.g: `pow ffs config <cid>`

Reply:
I like that. Will update similar commands as well.